atest/testdata/running/skip/skip.robot

```diff
@@ -248,6 +248,20 @@ Skipped with --SkipOnFailure when Set Tags Used in Teardown
     Fail    Ooops, we fail!
     [Teardown]    Set Tags    skip-on-failure
 
+Skipped when test is tagged with robot:skip
+    [Documentation]    SKIP
+    ...    Test skipped since test is tagged with robot:skip.
+    [Tags]    robot:skip
+    
```

Comment:
Would be good to add something like `Fail    Not executed` to the body of the test. Empty test is an error situation and although apparently now test is skipped before that error occurs, this behavior could easily change in the future.